kernel/src/actions/visitors.rs

```diff
@@ -25,6 +31,13 @@ impl MetadataVisitor {
         id: String,
         getters: &[&'a dyn GetData<'a>],
     ) -> DeltaResult<Metadata> {
+        require!(
+            getters.len() == 9,
```

Comment:
They aren't repeated, so naming them would just add cognitive overhead to chase down a definition?

(they're basically like unit tests -- we _want_ them to be very obviously hard-wired so that if anything breaks somebody has to go look at it)

Reply:
makes sense